README.md

```diff
@@ -124,13 +132,13 @@ The following specifications are relevant here:
 
 **Additional REQUIRED fields per `pl:item_type`:**
 
-| Field Name     | PSOrthoTile | PSScene | REOrthoTile | REScene | SkySatCollect | SkySatScene | SkySatVideo |
-| -------------- | ----------- | ------- | ----------- | ------- | ------------- | ----------- | ----------- |
-| instruments    | ✓ | ✓ |   |   |   |   |   |
-| gsd            | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |   |
-| eo:cloud_cover | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |   |
-| eo:snow_cover  |   | ✓ |   |   | ✓ | ✓ |   |
-| view:azimuth   | ✓ | ✓ |   |   | ✓ | ✓ | ✓ |
+| Field Name     | PSOrthoTile | PSScene | REOrthoTile | REScene | SkySatCollect | SkySatScene | SkySatVideo | Landsat8L1G | M(O/Y)D09G(A/Q) | Sentinel1 | Sentinel2L1C |
+| -------------- | ----------- | ------- | ----------- | ------- | ------------- | ----------- | ----------- | ----------- | --------------- | --------- | ------------ |
+| instruments    | ✓ | ✓ |   |   |   |   |   | ✓ | ✓ |   | ✓ |
+| gsd            | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |   | ✓ | ✓ | ✓ | ✓ |
+| eo:cloud_cover | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |   | ✓ | ✓ | ✓ | ✓ |
+| eo:snow_cover  |   | ✓ |   |   | ✓ | ✓ |   |   |   |   |   |
+| view:azimuth   | ✓ | ✓ |   |   | ✓ | ✓ | ✓ |   |   |   |   |
```

Comment:
Do you think the sun geometry properties should be included here too (`view:sun_elevation` and `view:sun_azimuth`)?